schema/manifestlist_test.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2016 The Linux Foundation
```

Comment:
This looks like it will conflict with your in-flight #444 (also adding a `schema/manifestlist_test.go` from scratch).